### PR TITLE
Serverinfo Command

### PR DESCRIPTION
**Pull request by ToothyDev**
Server info command, including:
- Owner
- Member Count
- Role Count
- Verification Level
- Channel Count
- Creation Date
- Emoji & Stickers Count
- Server Features
- Server ID
- Server Icon as thumbnail
- Server banner as image, if exists

### CODE REVIEW

File: cogs/utility.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Channels:** {len(guild.text_channels)} Text, {len(guild.voice_channels)} Voice
+**Created:** <t:{round(guild.created_at.timestamp())}:R>
+**Emojis:** {len(guild.emojis)}
+**Stickers:** {len(guild.stickers)}
+        """
+        embed.set_thumbnail(url=guild.icon.url)
+        embed.set_footer(text=f"ID: {guild.id}")
+        if (guild.banner):
+            embed.set_image(url=guild.banner.url)
+        for feature in guild.features:
```
Comment (JoshuaSlui) [inline cogs/utility.py]: A for loop isn't really necessary. You can just use `", ".join(guild.features)`

File: cogs/utility.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import discord
+from discord import slash_command
+from discord.ext import commands
+
+
+class Utility(commands.Cog, name="utility"):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @slash_command(brief="Information about the server")
```
Comment (JoshuaSlui) [inline cogs/utility.py:10]: Please remember to add """ descriptions """. Briefs don't actually add them correctly

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

File: cogs/utility.py (hunk 1 above)
Comment (ToothyDev) [reply cogs/utility.py]: Actually didn't know this, the more you know - was already wondering if there was some more elegant way of doing it

File: cogs/utility.py (hunk 2 above)
Comment (ToothyDev) [reply cogs/utility.py:10]: Are briefs necessary at all right now?

Comment (JoshuaSlui) [reply cogs/utility.py:10]: Not really.

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**